app/components/form-balancer-rules/component.js

```diff
@@ -34,6 +37,9 @@ export default Ember.Component.extend({
     protos.sort();
     this.set('protocolChoices', protos);
 
+    const regions = this.get('regions').get('all');
+    this.set('hasRegion', regions.content.length > 0)
```

Comment:
`regions.get('length')`.. Store returns an ArrayProxy, you never deal with the `content` of it directly.